source/developers/projects/studio/api/publish/publish-items.rst

```diff
@@ -25,17 +25,19 @@ Resource Information
 Parameters
 ----------
 
-+---------------+-------------+---------------+--------------------------------------------------+
-|| Name         || Type       || Required     || Description                                     |
-+===============+=============+===============+==================================================+
-|| site         || String     || |checkmark|  || Site to use                                     |
-+---------------+-------------+---------------+--------------------------------------------------+
-|| environment  || String     || |checkmark|  || Environment to publish to                       |
-+---------------+-------------+---------------+--------------------------------------------------+
-|| schedule     || Datetime   ||              || Schedule when to publish                        |
-+---------------+-------------+---------------+--------------------------------------------------+
-|| item         || String     || |checkmark|  || Path to the item(s)                             |
-+---------------+-------------+---------------+--------------------------------------------------+
++---------------------+-------------+---------------+--------------------------------------------------+
+|| Name               || Type       || Required     || Description                                     |
++=====================+=============+===============+==================================================+
+|| site               || String     || |checkmark|  || Site to use                                     |
```

Comment:
Should we change this one too? Should we support both site and site_id for backward compatibility?